Add PPRV cloud docs for public beta (#446)
hashicorp/terraform-docs-common: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

To view and manage policies and policy sets, you must have [manage policy permissions](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policies) for your organization.

> ## Policy Checks vs Policy Evaluations

Reviewer: (could be wrong here) The general advice I've had from docs people is to avoid contractions so perhaps just be more descriptive.

```suggestion
## Differences between Policy Checks  and Policy Evaluations
```

Though policy checks and policy evaluations seem like synonyms, there is a fundamental difference between the policy check and policy evaluation workflow.

### Policy Checks

Sentinel policies are currently run as a policy check. This is the default workflow for Sentinel.

> Policy checks are pinned to a single runtime version of Sentinel and have access to cost estimation data.

Reviewer 2: ```suggestion
Policy checks use the latest version of the Sentinel runtime and have access to cost estimation data.
```

### Policy Evaluations

> Policy evaluations are run within the [Terraform Cloud Agent](/terraform/cloud-docs/agents).

Reviewer 2: @mrinalirao do we need to explicit in this line and call out that evaluations are run within the agent, on terraform infrastructure just so there is no confusion about self-hosted agents?

> OPA policies run as policy evaluations. Policy evalutations do not have access to cost estimation data.

Reviewer 2: ```suggestion
Policy evaluations do not have access to cost estimation data.
```

> -> **Note:** Sentinel policy evaluations is currently in beta.

Reviewer: grammar nit: Should be

```suggestion
-> **Note:** Sentinel policy evaluations are currently in beta.
```

or

```suggestion
-> **Note:** Sentinel policy evaluation is currently in beta.
```

1. Choose a policy set scope:
   - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.
1. **(Sentinel Only) Choose a policy set type:

>    - **Standard:** .

Reviewer 2: @mrinalirao should this include both **standard** and **enhanced**?

>    - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.

Reviewer 2: @mrinalirao this line looks like it has shifted and should go above in the `choose a policy set scope` section

1. Choose a policy set scope:
   - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.

1. **(Sentinel Only)** Choose a policy set type:

>    - **Standard:** This is the default workflow. Sentinel policy set is run as a policy check in Terraform Cloud.
>    - **Enhanced:** Sentinel policy set is run as a policy evaluation in Terraform Cloud.

Reviewer: Suggestion: Maybe hyperlink `policy check` and `policy evaluation` if needed? maybe not

```suggestion
   - **Standard:** This is the default workflow. A Sentinel policy set uses a policy check in Terraform Cloud.
   - **Enhanced:** A Sentinel policy set uses a policy evaluation in Terraform Cloud.
```

The policy set no longer appears on the UI and Terraform Cloud no longer applies it to any workspaces. For managed policy sets, all of the individual policies are still available in Terraform Cloud. You must delete each policy individually to remove it from your organization.

> ### Pinning policy tool versions to policy sets

Reviewer 2: @mrinalirao Having reviewed the page outline, I think it may be advisable to place this into it's own section. What do you think?

> -> **Note:** This feature is in beta.

Reviewer 2: ```suggestion
-> **Note:** Policy runtime version management is currentl in beta.
```

You can pin a policy set to a specific version of the tool. This provides control over the versioning of the policy-as-code runtime.

#### OPA Policy sets

For OPA Policy sets, a `Runtime version` drop-down selector is available that allows you to choose supported versions of OPA. All policies within this policy set will be evaluated against the chosen policy tool version.

#### Sentinel Policy sets

> For Sentinel Policy sets, policy tool version pinning is available on an opt-in basis. Sentinel policy sets will default to using the policy check flow and you will be able to opt-in to the policy evaluation flow on a per-policy set basis through a radio button living on the Sentinel policy set configuration form.

Reviewer 2: ```suggestion
For Sentinel Policy sets, policy runtime version selection is available on an opt-in basis. Sentinel policy sets will default to using the policy check flow, and you can opt in to the policy evaluation flow for each policy set individually through a radio button located in the Sentinel policy set configuration form.
```

> After opting in to the `enhanced` workflow, you can choose the Sentinel runtime version (via the a `Runtime version` drop-down selector) against which the policies in that particular policy set will be evaluated.

Reviewer 2: ```suggestion
After opting in to the 'enhanced' workflow, you can select the Sentinel runtime version (via the 'Runtime version' drop-down selector) against which the policies in that specific policy set will be evaluated.
```

> Opting in also means that the policy sets are now evaluated in the [Terraform Cloud agent](/terraform/cloud-docs/agents). These policy sets will also not have access to the cost estimation data.

Reviewer 2: ```suggestion

All policy sets that are configured to use the `enhanced` workflows will be evaluated in the [Terraform Cloud agent](/terraform/cloud-docs/agents). 

-> **Note:** Policy sets that use the `enhanced` workflow do not have access to cost estimation data.
```

The Sentinel Policy sets will also have an `override` checkbox (similar to OPA). Failed policies stop the run, but any user with [Manage Policy Overrides permission](/terraform/cloud-docs/users-teams-organizations/permissions#manage-policy-overrides) can override these failures and allow the run to complete.
Both soft-mandatory and hard-mandatory policies can be overriden.

> -> **Note:** Only Sentinel tool version of 0.23.0 and above and Terraform Cloud agent version 1.14.0 and above support Sentinel tool version pinning

Reviewer 2: ```suggestion
-> **Note:** Sentinel tool version pinning is supported only for Sentinel 0.23.0 and above, as well as Terraform Cloud agent versions 1.14.0 and above
```
